_pages/how-we-work/software.md

```diff
@@ -24,7 +24,7 @@ _Note: If you're looking for MS Office, see [this page]({{site.baseurl}}/office/
 
 ## Purchase new software
 
-If the software is not in the TTS inventory or GSA IT Catalog (or if there are no licenses available), the TTS Tech Portfolio requests that you use the [TTS Tech Portfolio General Request Form](https://forms.gle/wPrCXJqDgaksXRwV8) for a _New Software (consult)_.
+If the software is not in the TTS inventory or GSA IT Catalog (or if there are no licenses available), the TTS Tech Portfolio requests that you first submit a [GSA IT Help Desk ticket](https://servicedesk.gsa.gov), you can also start a ticket by emailing `itservicedesk@gsa.gov` and provide us with the ticket number in Slack [#tts-tech-portfolio channel](https://gsa-tts.slack.com/archives/CNW3GL70S) or by email `devops@gsa.gov`.
```

Comment:
This is something that will presumably change soon, but for now we probably want them to reach out to our team, right? cc @its-a-lisa-at-work

Reply:
I made this pull request last night https://github.com/18F/handbook/pull/2091